mpf/config_players/plugin_player.py

```diff
@@ -50,30 +53,36 @@ def register_player_events(self, config, mode=None, priority=0):
 
         self.bcp_client = self._get_bcp_client(config)
 
+        m = self.machine
```

Comment:
Ah, okay. I'll make sure those settings get used. That should eliminate a lot of the messages on my side.